src/dymaptic.GeoBlazor.Core/Components/Symbols/MapFont.cs

```diff
@@ -38,7 +37,7 @@ public MapFont()
     /// <param name="weight">
     ///     The text weight.
     /// </param>
-    public MapFont(int? size, string? family, string? style, string? weight)
+    public MapFont(Dimension? size, string? family, string? style, string? weight)
```

Comment:
`Dimension` or `double`? for size... looking at the other `size` references below

Reply:
`Dimension` for the C# class. We _serialize_ this in Protobuf as a `double`. Notice at the bottom of the file you are looking at `MapFontSerializationRecord` (protobuf representation).